sdk/schemaregistry/azure-schemaregistry/azure/schemaregistry/_operations/_patch.py

```diff
@@ -0,0 +1,294 @@
+from azure.core.tracing.decorator import distributed_trace
+
+from ._operations import ClsType, SchemaRegistryClientOperationsMixin as GeneratedClientOperationsMixin
+from .._serialization import Serializer
+
+if TYPE_CHECKING:
+    from azure.core.pipeline import PipelineResponse
+
+_SERIALIZER = Serializer()
+
+
+def build_schema_registry_get_schema_id_by_content_request( # pylint: disable=name-too-long
+    group_name: str,
+    name: str,
+    **kwargs: Any
+) -> HttpRequest:
+    _headers = case_insensitive_dict(kwargs.pop("headers", {}) or {})
+    _params = case_insensitive_dict(kwargs.pop("params", {}) or {})
+
+    content_type: str = kwargs.pop('content_type')
+    api_version: str = kwargs.pop('api_version', _params.pop('api-version', "2022-10"))
+    accept = _headers.pop('Accept', "application/json")
+
+    # Construct URL
+    _url = "/$schemaGroups/{groupName}/schemas/{name}:get-id"
+    path_format_arguments = {
+        "groupName": _SERIALIZER.url("group_name", group_name, 'str'),
+        "name": _SERIALIZER.url("name", name, 'str'),
+    }
+
+    _url: str = _url.format(**path_format_arguments)  # type: ignore
+
+    # Construct parameters
+    _params['api-version'] = _SERIALIZER.query("api_version", api_version, 'str')
+
+    # Construct headers
+    _headers['Content-Type'] = _SERIALIZER.header("content_type", content_type, 'str')
+    _headers['Accept'] = _SERIALIZER.header("accept", accept, 'str')
+
+    return HttpRequest(
+        method="POST",
+        url=_url,
+        params=_params,
+        headers=_headers,
+        **kwargs
+    )
+
+def build_schema_registry_register_schema_request(  # pylint: disable=name-too-long
+    group_name: str,
+    name: str,
+    **kwargs: Any
+) -> HttpRequest:
+    _headers = case_insensitive_dict(kwargs.pop("headers", {}) or {})
+    _params = case_insensitive_dict(kwargs.pop("params", {}) or {})
+
+    content_type: str = kwargs.pop('content_type')
+    api_version: str = kwargs.pop('api_version', _params.pop('api-version', "2022-10"))
+    accept = _headers.pop('Accept', "application/json")
+
+    # Construct URL
+    _url = "/$schemaGroups/{groupName}/schemas/{name}"
+    path_format_arguments = {
+        "groupName": _SERIALIZER.url("group_name", group_name, 'str'),
+        "name": _SERIALIZER.url("name", name, 'str'),
+    }
+
+    _url: str = _url.format(**path_format_arguments)  # type: ignore
+
+    # Construct parameters
+    _params['api-version'] = _SERIALIZER.query("api_version", api_version, 'str')
+
+    # Construct headers
+    _headers['Content-Type'] = _SERIALIZER.header("content_type", content_type, 'str')
+    _headers['Accept'] = _SERIALIZER.header("accept", accept, 'str')
+
+    return HttpRequest(
+        method="PUT",
+        url=_url,
+        params=_params,
+        headers=_headers,
+        **kwargs
+    )
+
+class SchemaRegistryClientOperationsMixin(GeneratedClientOperationsMixin):
+    """
+    Implement custom operations.
```

Comment:
Are these custom operations here and not in the tsp because of the issue with multi-content type?

Reply:
So these ops are in the tsp. However, they don't generate b/c the generation doesn't currently support multiple content types in the request.

Reply:
so yeah, what you said lol